File: src/src/HIS.Application.Contracts/Inventory/Dtos/SupplierDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace HIS.Inventory.Dtos;

public class SupplierDto : AuditedEntityDto<Guid>
{
    public string Name { get; set; }
    public string ContactPerson { get; set; }
    public string Phone { get; set; }
    public string Email { get; set; }
    public string Address { get; set; }
    public string TaxId { get; set; }
}

File: src/angular/src/app/proxy/inventory/dtos/supplier-dto.ts
import { AuditedEntityDto } from '@abp/ng.core';

export interface SupplierDto extends AuditedEntityDto<string> {
    name: string;
    contactPerson: string;
    phone: string;
    email: string;
    address: string;
    taxId: string;
}

export interface CreateUpdateSupplierDto {
    name: string;
    contactPerson?: string;
    phone?: string;
    email?: string;
    address?: string;
    taxId?: string;
}
